Gemfile.lock

```diff
@@ -855,3 +855,9 @@ DEPENDENCIES
   webpacker (~> 5.4)
   webpush!
   xorcist (~> 1.1)
+
+RUBY VERSION
+   ruby 3.0.4p208
+
+BUNDLED WITH
+   2.2.33
```

Comment:
Probably want to drop this unrelated change.

Reply:
Know of any simple way of how to block that on bundle install? I keep having to manually remove it (and forgot this time). There are some solutions out there with modifying some bundler source files, but don't really wanna go there if there is a simple config or approach for it.